### PR TITLE
Major Update alias.js (w/ slight organization)

### PR DESCRIPTION
**Pull request by Yoshizilla315**
A lot of these I found were fairly common and surprised they weren't there. Some Pokemon have multiple because I've seen them spelled differently. If you have any questions about some, just ask.

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -505,6 +507,7 @@ exports.BattleAliases = {
 	"bdrill": "Beedrill",
 	"bulu": "Tapu Bulu",
 	"cathy": "Trevenant",
+	"chandy": "Chandelure",
```
Comment (DeathlyPlays) [inline data/aliases.js]: Add Chandie also tbh

Comment (Yoshizilla315) [inline data/aliases.js]: I've never heard that before.

Comment (DeathlyPlays) [inline data/aliases.js]: I have lol

Comment (Yoshizilla315) [inline data/aliases.js]: Interesting, if I start hearing it, I might add it in the future but who knows.

Comment (AustinXII) [inline data/aliases.js]: Uh have you read it being used before? I'm not trying to be rude but that just seems like you're miss typing Chandy after hearing it used

Comment (Yoshizilla315) [inline data/aliases.js]: I agree, I just don't think one would spell it that way.

Comment (AustinXII) [inline data/aliases.js]: Basically I agree with Yoshi with Chandie not being used

Comment (DeathlyPlays) [inline data/aliases.js]: Mkay

**Comment by Zarel** [comment]
Once again, someone else needs to approve aliases; I won't

**Comment by AustinXII** [comment]
Reuni, pert, mamo, darm, chandy, and maybe pex are the only ones I see useable

**Comment by Yoshizilla315** [comment]
Hm not Amoon? I never see people spell out Amoonguss...

**Comment by Zarel** [comment]
Unmaintained.

**Pull request closed without merging**